ksp_plugin/plugin.cpp

```diff
@@ -656,32 +656,33 @@ void Plugin::FreeVesselsAndPartsAndCollectPileUps(Time const& Δt) {
 
 void Plugin::SetPartApparentRigidMotion(
     PartId const part_id,
-    RigidMotion<RigidPart, World> const& rigid_motion) {
-  // As a reference frame, |ApparentBubble| differs from |World| only by having
-  // the same axes as |Barycentric| and being nonrotating.  However, there is
+    RigidMotion<RigidPart, ApparentWorld> const& rigid_motion) {
+  // As a reference frame, |Apparent| differs from |World| only by having the
+  // same axes as |Barycentric| and being nonrotating.  However, there is
   // another semantic distinction: |Apparent...| coordinates are uncorrected
   // data from the game, given immediately after its physics step; before using
   // them, we must correct them in accordance with the data computed by the pile
   // up.  This correction overrides the origin of position and velocity, so we
   // need not worry about the current definition of
   // |{World::origin, World::unmoving}| as we do when getting the actual degrees
   // of freedom (via |Plugin::BarycentricToWorld|).
-  RigidMotion<World, ApparentBubble> world_to_apparent_bubble{
-      RigidTransformation<World, ApparentBubble>{
-          World::origin,
-          ApparentBubble::origin,
-          OrthogonalMap<Barycentric, ApparentBubble>::Identity() *
-              renderer_->WorldToBarycentric(PlanetariumRotation())},
-      renderer_->BarycentricToWorld(PlanetariumRotation())(
-          -angular_velocity_of_world_),
-      World::unmoving};
+  RigidMotion<ApparentWorld, Apparent> world_to_apparent{
+      RigidTransformation<ApparentWorld, Apparent>{
+          ApparentWorld::origin,
+          Apparent::origin,
+          OrthogonalMap<Barycentric, Apparent>::Identity() *
+              renderer_->WorldToBarycentric(PlanetariumRotation()) *
+              OrthogonalMap<World, ApparentWorld>::Identity()},
+      Identity<World, ApparentWorld>()(renderer_->BarycentricToWorld(
+          PlanetariumRotation())(-angular_velocity_of_world_)),
```

Comment:
This `-` sign is frightening.  I wonder if we could hide it in a linear transformation (what it probably does is to convert the angular velocity of A wrt B into the angular velocity of B wrt A).

Reply:
Yeah this is the angular velocity *of* world, but we are giving the angular velocity *in* world (of apparent which rotates like barycentric).

I think now we have more constructors for RigidTransformation, maybe we can use the angular velocity of world more directly.